src/components/Button/index.js

```diff
@@ -7,8 +7,8 @@ const sizes = ['large', 'small', 'extra-large']
 const types = ['success', 'warning', 'danger', 'primary', 'default']
 
 const sizedBtnClasses = {
-  'large': 'h3 pa3',
-  'small': 'h2 lh-solid pa3',
+  'large': 'pa3',
+  'small': 'lh-solid pa3',
```

Comment:
@DrShpongle We removed static/hard-coded styles like this so everything is fluid/responsive. In the future, it would be good to avoid hard-coded pixels for things like heights, widths, and positioning.